src/css/base.scss

```diff
@@ -2424,6 +2424,10 @@ textarea,
     padding-left: 16px !important;
 }
 
+.pl-8 {
+    padding-left: 8px;
```

Comment:
Why did you create this pl-8 it can be used directly as it is written in formula.scss

Reply:
no need to create css for this

Reply:
Removed it.